File: Exemplos/App_IntentDigitalHUB_XamarinAndroid/Elgin_IntentDigitalHUB_XamarinAndroid/IntentServices/Termica/AvancaPapel.cs
namespace Xamarin_Android_Intent_Digital_Hub.IntentServices.Termica
{
    class AvancaPapel : TermicaCommand
    {
        readonly int linhas;

        public AvancaPapel(int linhas) : base("AvancaPapel")
        {
            this.linhas = linhas;
        }

        protected override string FunctionParameters()
        {
            return "\"linhas\"" + ":" + linhas;
        }
    }
}

File: Exemplos SmartPOS/App_IntentDigitalHUB_ReactNative/Elgin_IntentDigitalHUB_SmartPOS_ReactNative-font/src/intentDigitalHubService/TERMICA/Commands/AvancaPapel.ts
import {TermicaCommand} from '../TermicaCommand';

export class AvancaPapel extends TermicaCommand {
  readonly linhas: number;

  constructor(linhas: number) {
    super('AvancaPapel');
    this.linhas = linhas;
  }

  functionParameters(): object {
    return {linhas: this.linhas};
  }
}
